projects/aws-containers/hello-eks-anywhere/helm/schema.json

```diff
@@ -4,11 +4,33 @@
   "title": "hello-eks-anywhere",
   "type": "object",
   "properties": {
+    "namespace": {
+      "type": "string",
+      "default": "default",
+      "description": "Namespace for package and resources."
+    },
     "sourceRegistry": {
       "type": "string",
       "default": "public.ecr.aws/eks-anywhere",
       "description": "Source registry for package."
     },
+    "repository": {
+      "type": "string",
+      "default": "hello-eks-anywhere",
+      "description": "Repository for package."
+    },
+    "imagePullSecrets": {
+      "type": "array",
+      "default": [],
+      "description": "Container imagePullSecrets.",
+      "items": {
+        "type": "object"
+      }
+    },
+    "tag": {
+      "type": "string",
+      "description": "Container image tag."
+    },
```

Comment:
Not sure we'd want to expose this or repository. Kind of goes against what were are trying to do here. No doubt we do in other packages, but I would argue we shouldn't. With either of these in there, someone that only had permissions to create packages resources could easily get access to other resources.